<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="interperter_path_lineEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>python解释器路径</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="4,1,2">
      <property name="spacing">
       <number>10</number>
      </property>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>下载</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="download_button">
        <property name="text">
         <string>下载</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="3,1">
      <item row="1" column="0">
       <widget class="QLineEdit" name="query_line_edit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QTableWidget" name="package_table"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="delete_button">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>617</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>虚拟环境</string>
    </property>
    <addaction name="create_venv"/>
    <addaction name="open_venv"/>
   </widget>
   <widget class="QMenu" name="menupip">
    <property name="title">
     <string>pip位置</string>
    </property>
    <addaction name="pip_download_source"/>
   </widget>
   <widget class="QMenu" name="menupython">
    <property name="title">
     <string>pyinstaller</string>
    </property>
    <addaction name="export_exe_action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menupip"/>
   <addaction name="menupython"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="create_venv">
   <property name="text">
    <string>创建虚拟环境</string>
   </property>
  </action>
  <action name="open_venv">
   <property name="text">
    <string>打开虚拟环境</string>
   </property>
  </action>
  <action name="pip_download_source">
   <property name="text">
    <string>下载镜像位置</string>
   </property>
  </action>
  <action name="export_exe_action">
   <property name="text">
    <string>py文件打包</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
